quaddtype/numpy_quaddtype/src/casts.cpp

```diff
@@ -771,7 +844,11 @@ add_cast_from(PyArray_DTypeMeta *to)
             .name = "cast_QuadPrec_to_NumPy",
             .nin = 1,
             .nout = 1,
+#if NPY_FEATURE_VERSION > NPY_2_3_API_VERSION
+            .casting = NPY_SAME_VALUE_CASTING,
+#else
             .casting = NPY_UNSAFE_CASTING,
+#endif
```

Comment:
I am not clear how this is used, doesn't the resolver actually state which casting is supported?